build.py

```diff
@@ -166,6 +203,12 @@
         # Already in a virtual environment, use current Python
         python_bin = sys.executable
 
+    # copy the process env vars
+    pip_env: dict[str, str] = {**os.environ}
```

Comment:
Looking around, seems `pip-env = os.environ.copy()` might be more canonical (and a little more intuitive and readable)

Reply:
`{**os.environ}` is what we used in the qdk, but `copy()` works too.